npm-shrinkwrap.json

```diff
@@ -0,0 +1,9532 @@
+      "integrity": "sha1-S8A3PBZLwykbTTaMgpzxqApZ3KA=",
+      "dev": true,
+      "requires": {
+        "map-visit": "^1.0.0",
+        "object-visit": "^1.0.0"
+      }
+    },
+    "color-convert": {
+      "version": "1.9.3",
+      "resolved": "https://registry.npmjs.org/color-convert/-/color-convert-1.9.3.tgz",
+      "integrity": "sha512-QfAUtd+vFdAtFQcC8CCyYt1fYWxSqAiK2cSD6zDB8N3cpsEBAvRxp9zOGg6G/SHHJYAT88/az/IuDGALsNVbGg==",
+      "dev": true,
+      "requires": {
+        "color-name": "1.1.3"
+      }
+    },
+    "color-name": {
+      "version": "1.1.3",
+      "resolved": "https://registry.npmjs.org/color-name/-/color-name-1.1.3.tgz",
+      "integrity": "sha1-p9BVi9icQveV3UIyj3QIMcpTvCU=",
+      "dev": true
+    },
+    "color-support": {
+      "version": "1.1.3",
+      "resolved": "https://registry.npmjs.org/color-support/-/color-support-1.1.3.tgz",
+      "integrity": "sha512-qiBjkpbMLO/HL68y+lh4q0/O1MZFj2RX6X/KmMa3+gJD3z+WwI1ZzDHysvqHGS3mP6mznPckpXmw1nI9cJjyRg==",
+      "dev": true
+    },
+    "combined-stream": {
+      "version": "1.0.8",
+      "resolved": "https://registry.npmjs.org/combined-stream/-/combined-stream-1.0.8.tgz",
+      "integrity": "sha512-FQN4MRfuJeHf7cBbBMJFXhKSDq+2kAArBlmRBvcvFE5BB1HZKXtSFASDhdlz9zOYwxh8lDdnvmMOe/+5cdoEdg==",
+      "dev": true,
+      "requires": {
+        "delayed-stream": "~1.0.0"
+      }
+    },
+    "commander": {
+      "version": "2.8.1",
+      "resolved": "https://registry.npmjs.org/commander/-/commander-2.8.1.tgz",
+      "integrity": "sha1-Br42f+v9oMMwqh4qBy09yXYkJdQ=",
+      "dev": true,
+      "optional": true,
+      "requires": {
+        "graceful-readlink": ">= 1.0.0"
+      }
+    },
+    "component-emitter": {
+      "version": "1.3.0",
+      "resolved": "https://registry.npmjs.org/component-emitter/-/component-emitter-1.3.0.tgz",
+      "integrity": "sha512-Rd3se6QB+sO1TwqZjscQrurpEPIfO0/yYnSin6Q/rD3mOutHvUrCAhJub3r90uNb+SESBuE0QYoB90YdfatsRg==",
+      "dev": true
+    },
+    "concat-map": {
+      "version": "0.0.1",
+      "resolved": "https://registry.npmjs.org/concat-map/-/concat-map-0.0.1.tgz",
+      "integrity": "sha1-2Klr13/Wjfd5OnMDajug1UBdR3s=",
+      "dev": true
+    },
+    "concat-stream": {
+      "version": "1.6.2",
+      "resolved": "https://registry.npmjs.org/concat-stream/-/concat-stream-1.6.2.tgz",
+      "integrity": "sha512-27HBghJxjiZtIk3Ycvn/4kbJk/1uZuJFfuPEns6LaEvpvG1f0hTea8lilrouyo9mVc2GWdcEZ8OLoGmSADlrCw==",
+      "dev": true,
+      "requires": {
+        "buffer-from": "^1.0.0",
+        "inherits": "^2.0.3",
+        "readable-stream": "^2.2.2",
+        "typedarray": "^0.0.6"
+      },
+      "dependencies": {
+        "isarray": {
+          "version": "1.0.0",
+          "resolved": "https://registry.npmjs.org/isarray/-/isarray-1.0.0.tgz",
+          "integrity": "sha1-u5NdSFgsuhaMBoNJV6VKPgcSTxE=",
+          "dev": true
+        },
+        "process-nextick-args": {

```

Comment:
Based on the console output of my local build `gulp-sass@2.3.2` (see [node-sass-compilation.txt](https://github.com/HistoryAtState/hsg-shell/files/3402540/node-sass-compilation.txt)) appears to a dependency a version of `node-sass` not available as a pre-built binary for macOS (the console log references v3.13.1, and the release matrix at https://github.com/sass/node-sass/releases/tag/v3.13.1 confirms that there's no support for 7`). As a result, there's a rather lengthy local compilation that happens, as the log above shows. The first release of node-sass that officially supports `node@10` is v4.9.0 (see https://github.com/sass/node-sass/releases/tag/v4.9.0). Would it be possible to upgrade us to a version of `gulp-sass` that is compatible with `node@10`? Reading the release notes for `gulp-sass` at https://github.com/dlmanning/gulp-sass/releases, I believe v4.0.0 would do the trick? 

The current release of `gulp-sass`, 4.0.2, depends on `node-sass^4.8.3`, which according to https://semver.npmjs.com/ means it would install `node-sass@4.9.0` or newer, which means we'd be compatible with `node@10`.

Reply:
@plutonik-a I've just added commits that address the `gulp-sass` and related `node-sass` issues. I've confirmed that the package builds without errors (or unexpected local compilations of dependencies), and the resulting hsg-shell xar installs fine... but there may be some sass issues. Namely, the previous/next navigation buttons when viewing FRUS documents appear as squares instead of arrows:

> ![Screen Shot 2019-07-17 at 1 32 15 PM](https://user-images.githubusercontent.com/59118/61397208-5d476980-a897-11e9-9898-8bc82da7eaed.png)

Perhaps hsg-shell's sass usage isn't compatible with this version of gulp-sass?

Reply:
@joewiz Thanks for spotting this! The problem here are my changes to the gulpfile: Due to the new gulp 4 syntax I had to apply changes in the task stream (`parallel` and `series`), which I obviously messed up at some point: The routing of task `clean` and copying the fonts is not correct with the effect, that the bootstrap font folder is deleted after being copied. Will fix this.

Reply:
@plutonik-a Excellent. Thanks in advance!